### PR TITLE
Airbrake backend: Support the new backend system

### PR DESCRIPTION
**Pull request by anicke**
## Hey, I just made a Pull Request!

Added support for the new backend system for the airbrake plugin.

#### :heavy_check_mark: Checklist

- [X] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by anicke** [comment]
Ping #18301

File: plugins/airbrake/README.md [hunk 1]
```diff
@@ -98,6 +98,17 @@ The Airbrake plugin provides connectivity between Backstage and Airbrake (https:
    }
    ```
 
+   ```typescript
```
Comment (awanlin) [inline plugins/airbrake/README.md:101]: We aren't ready to have everyone use the new backend system as the default. You'll want to keep the existing documentation as is. But then add a new section specifically about this. The Azure DevOps Backend is a good example: https://github.com/backstage/backstage/tree/master/plugins/azure-devops-backend#new-backend-system

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for the contribution @anicke 🚀 

The code changes look good to me just left a comment about the `README`

**Comment by anicke** [comment]
Made the change on my phone. Could not figure out how to sign the commit. Will fix later.

**Review by awanlin — APPROVED** [review]
Looks good to me, thanks! 🚀 

Just make sure to resolve the issue with the DCO.

FYI - The build failures do not looks related and we are seeing them fail the same way for other PRs

**Comment by benjdlambert** [comment]
@anicke we also need a changeset for the readme update in the `airbrake` plugin :pray:

**Comment by anicke** [comment]
Changeset added!

**Review by benjdlambert — APPROVED** [review]
Nice thanks :tada:

**Pull request merged**